src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,117 @@
+@import "../utils/mixins";
+@import "../utils/vars";
+
+.page__body--with-menu {
+  overflow: hidden;
+}
+
+.menu {
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  grid-template-rows: repeat(4, max-content);
+  z-index: 2;
+
+  background-color: #e8eff9;
+  transform: translateX(100%);
+  transition: transform $animation-duration;
+  overflow-y: auto;
+
+  @include set-horizontal-padding;
+
+  @include set-smartphone-grid;
+
+  @include for-tablet {
+    @include set-tablet-grid;
+  }
+
+  @include for-desktop {
+    @include set-desktop-grid;
+  }
+
+  &:target {
+    transform: translateX(0);
+  }
+
+  &__ {
+    &close {
+      width: 16px;
+      height: 16px;
+      grid-row: 1;
+      grid-column: -1 / -2;
+      justify-self: flex-end;
+      margin-top: 40px;
+
+      @include for-tablet {
+        margin-top: 56px;
+      }
+
+      @include for-desktop {
+        width: 21px;
+        height: 21px;
+        margin-top: 54px;
+      }
+    }
+
+    &list {
+      margin-top: 23px;
+      grid-column: 1 / -1;
+      grid-row: 2;
+
+      @include for-tablet {
+        margin-top: 24px;
+      }
+
+      @include for-tablet {
```

Comment:
This media query should be '@include for-desktop' instead of '@include for-tablet', as it seems like you are targeting the desktop layout.

Reply:
I renamed it.